File: C/1013.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int A,B,C,Maiorab,Maior;
    scanf("%d %d  %d",&A,&B,&C);
    Maiorab=(A+B+abs(A-B))/2;
    Maior=(Maiorab+C+abs(Maiorab-C))/2;
    printf("%d ",Maior);
    printf("eh o maior\n");
    return 0;
}

File: Kotlin/1013.kt
fun main(args: Array<String>) {

  var a:Int = readLine()!!.toInt()
  var b:Int = readLine()!!.toInt()
  var c:Int = readLine()!!.toInt()

  var maior = a
  if ((a < b) || (a < c)){
      maior = if (b > c){
          b
      } else {
          c
      }
  }
  
  println("$maior eh o maior")
}